rewrite-core/src/main/java/org/openrewrite/PrintOutputCapture.java

```diff
@@ -67,6 +71,22 @@ public PrintOutputCapture<P> clone() {
     }
 
     public interface MarkerPrinter {
+
+        @Incubating(since = "8.41.4")
+        @RequiredArgsConstructor
+        enum MarkerMode {
+            NONE(MarkerPrinter.NONE),
+            DEFAULT(MarkerPrinter.DEFAULT),
+            VERBOSE(MarkerPrinter.VERBOSE),
+            FENCED(MarkerPrinter.FENCED),
```

Comment:
I think it would be nice if we can come up with slightly more self-explanatory names. NONE is clear, the others less so. Can we use MARKUP_ONLY (even if that includes SearchResult)?

Reply:
it would indeed be nice. Any more suggestions? :D
We could also add (javadoc) comments to explain the differences

Reply:
For FENCED we can maybe use something like MARKUP_IDS or similar.

Reply:
Not sure about the actual names. I suppose DEFAULT and VERBOSE are OK too.